packages/neon/neon/lib/src/widgets/relative_time.dart

```diff
@@ -1,49 +1,52 @@
+import 'dart:async';
+
 import 'package:flutter/widgets.dart';
+import 'package:neon/src/utils/relative_time.dart';
 
-class RelativeTime extends StatelessWidget {
+/// Shows the time elapsed since a [DateTime] and periodically updates itself.
+class RelativeTime extends StatefulWidget {
+  /// Creates a new relative DateTime widget.
```

Comment:
I find these constructor docs useless if they only say that they create a new object of the class. Better to ignore those as they are not helpful imo